packages/manager/apps/web/client/app/domain/contact/contact.routing.js

```diff
@@ -7,6 +9,13 @@ export default /* @ngInject */ ($stateProvider) => {
       },
     },
     resolve: {
+      goToReassignContactsUrl: /* @ngInject */ (coreURLBuilder) => (domain) =>
+        coreURLBuilder.buildURL('dedicated', '#/contacts/services/edit', {
+          serviceName: domain,
+          category: PRODUCT_TYPE,
+          service: domain,
```

Comment:
maybe we can get domain directly in state params url no need function we can return directly string during the resolve.